src/main/java/com/google/sps/data/ActivityEvent.java

```diff
@@ -2,16 +2,38 @@
 
 import java.util.List;
 
-public class ActivityEvent {
+public final class ActivityEvent {
```

Comment:
why do we need final here? Is it necessary?

Reply:
It's not necessary. It's just to say that the class can't be extended. It's good semantics though.

Reply:
+1 to this being a good practice.